aiocoap/interfaces.py

```diff
@@ -143,18 +143,22 @@ def scheme(Self):
         communication. (Should there ever be a scheme that addresses the
         participants differently, a scheme_local will be added.)"""
 
-    maximum_block_size_exp = MAX_REGULAR_BLOCK_SIZE_EXP
-    """The maximum negotiated block size that can be sent to this remote."""
+    @property
+    def maximum_block_size_exp(self) -> int:
```

Comment:
This is an API change, but as https://stackoverflow.com/questions/58349417/how-to-annotate-attribute-that-can-be-implemented-as-property elaborates, the API was already weird (instances can alter the value *unless it's one of the types where they can't*). To mitigate the impact, I'm adding regular maximum_block_size_exp attributes to those types where it was possible before.